Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -357,6 +398,25 @@ class NavigatorStatusObserver: NavigatorObserver {
     }
 }
 
+class NavigatorRouteAlternativesObserver: RouteAlternativesObserver {
+
+    func onRouteAlternativesChanged(for routeAlternatives: [RouteAlternative], removed: [RouteAlternative]) -> [NSNumber] {
+        let userInfo: [Navigator.NotificationUserInfoKey: Any] = [
+            .alternativesListKey: routeAlternatives,
+            .removedAlternativesKey: removed,
+        ]
+        NotificationCenter.default.post(name: .navigatorDidChangeAlternativeRoutes, object: nil, userInfo: userInfo)
+        return [] //notify NN that we didn't discard any of the route alternatives
```

Comment:
Do we want to use this return later? As I don't see how can we extend the API introduced in this PR to be able to return anything here.